design-system/react/src/components/PasswordStrength/PasswordStrength.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
+
+import { InputPassword } from '../InputPassword';
+import { Stack } from '../Stack';
+
+import type { PasswordStrengthProps } from './PasswordStrength';
+import { PasswordStrength } from './PasswordStrength';
+
+export default {
+  component: PasswordStrength,
+  title: 'Overlay/PasswordStrength',
+};
+
+export const Usage = (args: PasswordStrengthProps) => {
+  const [password, setPassword] = useState('');
+  const [open, setOpen] = useState(false);
+
+  return (
+    <Stack>
```

Comment:
Try to add a `width: 350px` here, it's to big the story preview
<img width="1108" alt="CleanShot 2022-11-17 at 21 17 13@2x" src="https://user-images.githubusercontent.com/2029172/202625481-3b0385ac-6d99-4a6e-836d-344b9e9a3d9b.png">